client/state/stats/lists/reducer.js

```diff
@@ -10,7 +10,7 @@ import { merge, unset } from 'lodash';
 import { createReducer } from 'state/utils';
 import { isValidStateWithSchema } from 'state/utils';
 import { getSerializedStatsQuery } from './utils';
-import { itemSchema, requestsSchema } from './schema';
```

Comment:
Will want to remove the exported schema from `./schema` since it's no longer used.